RockLib.UniversalMemberAccessor/UniversalMemberAccessor.cs

```diff
@@ -308,8 +311,17 @@ public override int GetHashCode()
         /// <returns>
         /// true if the operation is successful; otherwise, false.
         /// </returns>
-        public override bool TryGetMember(GetMemberBinder binder, out object result)
+#if NET48
+        public override bool TryGetMember(GetMemberBinder binder, out object? result)
```

Comment:
Wouldn't we want to check if binder is null in either case?

Just read about the [NotNull], interesting.  What situations would we want to add that?

Reply:
Ok, going back and looking at it I probably should just check for null.

Reply:
Should the other instances like this be changed to just check for the parameter being null and throwing as well?

Reply:
This was the only instance that was flagged with an error requiring it be checked, so I would think everything else is good.

Reply:
Yes, I missed that I did that in more than one place lol.